packages/components/src/internal/components/search/SampleFinderSection.tsx

```diff
@@ -0,0 +1,183 @@
+    parentEntityDataTypes: EntityDataType[],
+    onAddEntity: (entityType: EntityDataType) => void;
+}
+
+export const SampleFinderHeaderButtons: FC<SampleFinderHeaderProps> = memo(props => {
+    const { parentEntityDataTypes, onAddEntity } = props;
+
+    return (
+        <div>
+            Search by:
+            {parentEntityDataTypes.map((parentEntityType) => (
+                <button key={parentEntityType.nounSingular} className="btn btn-default margin-left" onClick={() => { onAddEntity(parentEntityType) }}>
+                    <i className="fa fa-plus-circle container--addition-icon" /> {capitalizeFirstChar(parentEntityType.nounAsParentSingular)} Properties
+                </button>
+            ))}
+        </div>
+    )
+});
+
+export const SampleFinderSection: FC<Props> = memo((props) => {
+    const { parentEntityDataTypes, ...gridProps } = props;
+
+    const [showFilterModal, setShowFilterModal] = useState<boolean>(false);
+    const [chosenEntityType, setChosenEntityType] = useState<EntityDataType>(undefined);
+    const [filterCards, setFilterCards] = useState<FilterCardProps[]>([]);
+
+    const onAddEntity = useCallback((entityType: EntityDataType) => {
+        setChosenEntityType(entityType);
+        setShowFilterModal(true);
+    }, []);
+
+    const onFilterEdit = useCallback((index: number) => {
+        console.log("onFilterEdit for index " + index + ": Not yet implemented.");
+        setShowFilterModal(true);
+    }, [filterCards]);
+
+    const onFilterDelete = useCallback((index: number) => {
+        const newFilterCards = [...filterCards];
+        newFilterCards.splice(index, 1);
+        setFilterCards(newFilterCards);
+    }, [filterCards]);
+
+    const onFilterClose = () => {
+        setShowFilterModal(false);
+    };
+
+    const onFind = useCallback((schemaQuery: SchemaQuery, filterArray: Filter.IFilter[])  => {
+        onFilterClose();
+        let newFilterCards = [...filterCards];
+        newFilterCards.push({
+            schemaQuery,
+            filterArray,
+            entityDataType: chosenEntityType,
+            index: filterCards.length,
+            onAdd: onAddEntity,
+            onDelete: onFilterDelete,
+        });
+        setFilterCards(newFilterCards);
+    }, [setFilterCards, filterCards, onFilterEdit, onFilterDelete, chosenEntityType]);
+
+    return (
+        <Section title={SAMPLE_FINDER_TITLE} caption={SAMPLE_FINDER_CAPTION} context={(
+            <SampleFinderHeaderButtons parentEntityDataTypes={parentEntityDataTypes} onAddEntity={onAddEntity}/>
+        )}
+        >
+            {filterCards.length == 0 ?
+                <>
+                    <FilterCards className="empty" cards={parentEntityDataTypes.map((entityDataType) => ({
+                        entityDataType,
+                        onAdd: onAddEntity
+                    }))} />
+                    <div className="filter-hint">{getFinderStartText(parentEntityDataTypes)}</div>
```

Comment:
minor: this is likely from the mockup, but this filter hint text size seems a little big to me when I look at it in the app